src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -28,4 +57,16 @@ export class BreadcrumbItemComponent {
 	@HostBinding("class.bx--breadcrumb-item--current") @Input() current = false;
 
 	@HostBinding("class.bx--breadcrumb-item") itemClass = true;
+
+	protected _href = "javascript:void(0)";
```

Comment:
This can be undefined, so we preserve the old behaviour
```suggestion
	protected _href;
```